ethportal-api/Cargo.toml

```diff
@@ -50,6 +51,11 @@ tokio = { version = "1.14.0", features = ["full"] }
 ureq = { version = "2.5.0", features = ["json"] }
 url = "2.3.1"
 validator = { version = "0.13.0", features = ["derive"] }
+shadow-rs = "0.27"
+const_format = {version = "0.2.0", features = ["rust_1_64"]}
+
+[build-dependencies]
+shadow-rs = "0.27"
```

Comment:
```suggestion
shadow-rs = "0.27"
const_format = {version = "0.2.0", features = ["rust_1_64"]}
```

Do we need shadow-rs in both build-dependcies and depends?

Reply:
Yeah, it collects the data in the build.rs and gives you access to it in the deps.

By the way, the const-format dep used indirectly (it is forwarded by the shadowrs, I declared it to get the split function that is under the feature gate)